ftrip.io.catalog-service/ftrip.io.catalog-service/Installers/DependenciesInstaller.cs

```diff
@@ -22,6 +22,7 @@ public void Install()
             _services.AddProxiedScoped<IAmenityRepository, AmenityRepository>();
             _services.AddProxiedScoped<IAmenityTypeRepository, AmenityTypeRepository>();
             _services.AddProxiedScoped<IPropertyTypeRepository, PropertyTypeRepository>();
+            _services.AddProxiedScoped<IBookingServiceClient, BookingServiceClient>();
```

Comment:
Http services should be added with "AddScoped". There is already built-in tracer that will open new tracing session.